clients/src/main/java/org/apache/kafka/clients/consumer/Consumer.java

```diff
@@ -106,7 +106,7 @@ public interface Consumer<K, V> extends Closeable {
     /**
      * @see KafkaConsumer#commitSync(Map, Duration)
      */
-    void commitSync(final Map<TopicPartition, OffsetAndMetadata> offsets, final Duration timeout);
+    void commitSync( Map<TopicPartition, OffsetAndMetadata> offsets,  Duration timeout);
```

Comment:
The extra whitespace between the parentheses and `Map` broke the checkStyle recommendations. Could you pls fix the same!